website/docs/opengl/c1/3-hello-texture.md

```diff
@@ -0,0 +1,462 @@
+```
+Enabling blending essentially tells OpenGL to select the output colors from the visible primitives according to some value. This is referred to as blending.
+In our case, we want to blend based on the alpha value.
+To configure this, we use the following line:
+```c#
+_gl.BlendFunc(BlendingFactor.SrcAlpha, BlendingFactor.OneMinusSrcAlpha);
+```
+
+The `BlendFunc` function configures the calculations that OpenGL does when blending output colors. In this case, we're telling it to use the alpha value of the sources
+being rendered (`BlendingFactor.SrcAlpha`) as the factor and to subtract that value from 1 to get the output value. With this, in most cases transparency will work exactly
+as you expect.
+
+If this isn't sufficient, there are a large amount of other blending function configurations you can use, which you can find in [the Khronos documentation](https://registry.khronos.org/OpenGL-Refpages/gl4/html/glBlendFunc.xhtml).
+
+And when you run the program now, the transparent pixels of the image will not be visible anymore:
+
+![Quad with texture and transparency](../../../images/opengl/chapter1/lesson3/quad-with-transparency.png)
+
+you can see the code final result [clicking here](../sources/1.3-final-result.html).
+
+## Texture parameters
+And now, some extra content to go with this tutorial!
+
+If you get curious about how the texture parameters work, let's learn it now!
+
+First, let's learn the structure of the command:
+```c#
+_gl.TexParameter( [Texture target] , [Parameter to change] , [New value for parameter] );
+// The suffix of TexParameter will vary depending on the type of the expected value for the parameter
+```
+
+The texture ID you already know. So let's learn what each parameter means!
+
+### `TextureWrapS` & `TextureWrapT`:
+`TextureWrap` is the parameter that allows you to say what the `texture` method (in the fragment shader) should do with values lesser than 0 and
+greater than 1. The `S` and `T` refer to X and Y respectively.
+
+Let's see the most common values for these parameters:
+
+#### `TextureWrapMode.Repeat`:
+Just repeats the image without any change, turning any coordinates outside the [0, 1) range by taking the fractional part of said number.  
+![Repeat](../../../images/opengl/chapter1/lesson3/texParameters/repeat.png)
+
+#### `TextureWrapMode.MirroredRepeat`:
+Mirror the texture for each 1 texture coordinate unit.  
+![Mirrored](../../../images/opengl/chapter1/lesson3/texParameters/repeatMirrored.png)
+
+#### `TextureWrapMode.ClampToEdge`:
+Returns the pixel on the respective edge of the image.  
+![A weird border](../../../images/opengl/chapter1/lesson3/texParameters/clampToEdge.png)
+
+### `TextureMinFilter` & `TextureMagFilter`:
+Texture `min` and `mag` filters are the filters used when the texture's final size is, respectively, lesser than or greater than the original size.
+For now, we will show just the main two options and in the next section [(Mipmaps)](#mipmaps), some others will be shown.
```

Comment:
```suggestion
### `TextureMinFilter` & `TextureMagFilter`:
When we draw a texture, the area on the screen in which we're drawing the texture typically doesn't have the same size or shape as the texture. This means that during rendering we usually have to either scale up or down the size of the texture! This process is called _filtering_.

The texture `min` (minifying) and `mag` (magnifying) filters are the filters used when the texture's final size is, respectively, lesser than or greater than the original size.
For now, we will show just the main two options and in the next section [(Mipmaps)](#mipmaps), some others will be shown.
```

Reply:
This suggestion applies to two previous conversations about missing explanations on min/mag filters